src/website/src/app/documentation/demos/buttons/buttons.demo.html

```diff
@@ -248,8 +245,7 @@ <h6 class="action-based-color">Action-based Color</h6>
                         button when “deleting” files indicates high severity.</p>
                 </div>
                 <div class="clr-col-md-6 clr-col-12">
-                    <img class="clrweb-Img" src="assets/images/documentation/buttons/action_colors.png"
-                        alt="Action colors can indicate severity"/>
+                    <img class="clrweb-Img" src="assets/images/documentation/buttons/action_colors.png" alt=""/>
```

Comment:
Should the alt text be removed here, I thought they were needed for screen readers?

Reply:
This is also related to the other empty alt tags below.

Reply:
They are but recommendation is if the image alt is describing text of near by content it is better to not have the alt as it reads the content twice. Chris explains it in more detail here https://github.com/vmware/clarity/issues/3181